corehq/apps/hqwebapp/templates/hqwebapp/partials/maintenance_alerts.html

```diff
@@ -0,0 +1,6 @@
+{% for alert in maintenance_alerts %}
+  <div class="alert alert-warning alert-maintenance hide" data-id="{{ alert.id }}">
+      <button class="close" data-dismiss="alert" aria-label="close">&times;</button>
+      {{ alert.html }}
```

Comment:
Confirming you tested locally that alerts with html in them still display properly?

Reply:
It's consistent with current alerts (as tested on staging) - URLs get wrapped in `<a>` tags to become links but anything else is rendered as plain text.